src/openseadragon.js

```diff
@@ -478,6 +478,10 @@
   *     Note: {@link OpenSeadragon.Options.showNavigationControl} is overriding
   *     this setting when set to false.
   *
+  * @property {Boolean} [showFlipControl=false]
```

Comment:
Looks like we also need a doc comment for the `flipped` option, right?

Reply:
It already has a doc comment below, or do you want something more complete?
(If true then the flip controls will be displayed as part of the standard controls.)